client/me/security-2fa-key/add.jsx

```diff
@@ -23,19 +25,24 @@ class Security2faKeyAdd extends React.Component {
 		registerRequests: PropTypes.object.isRequired,
 	};
 
+	state = {
+		error: false,
+	}
+	
 	componentDidMount = () => {
 		this.registerKey();
 	};
 
 	registerKey = () => {
+		this.setState( { error: false } );
 		webauthn
 			.register()
 			.then( data => {
 				debug( 'registered key with data', data );
 				this.keyRegistered();
 			} )
 			.catch( err => {
-				this.props.onCancel( err );
+				this.setState( { error: err } );
```

Comment:
Nit pick maybe we can just name `err` as `error` above so that we can just do:

```
this.setState( { error } );
```

I think this works in ES6?

Reply:
Fixed in f41e605 . Thanks for the tip, I had no idea this can be done!